ui-components/src/page_components/listing/ListingCard.tsx

```diff
@@ -130,7 +145,7 @@ const ListingCard = (props: ListingCardProps) => {
               <hr className={"mb-2"} />
             )}
           <div className={"listings-row_headers"}>
-            {getHeader(contentProps?.tableHeader, 4, "tableHeader")}
+            {getHeader(contentProps?.tableHeader, 4, "tableHeader", "", true)}
```

Comment:
If we flip the order of the two new props, we should be able to not have to pass the empty string